### ISSUE TITLE
Favicon doesn't show the Bazarr icon

### ISSUE THREAD

**Issue by yannduran** [morpheus65535/bazarr#1976]
**Describe the bug**
I'm not seeing Bazarr's favicon, it just shows Edge's default icon (see screenshot below).

I've cleared the Edge's `Cached images and files` restarted Edge. I still don't get the Bazarr icon, even using CTRL-F5.

**Screenshots**
<img width="45" alt="22-10-23_12-58-17" src="https://user-images.githubusercontent.com/2031126/197369698-1331021e-5e36-406d-baf8-d421518fadbf.png">

**Software**
 - Bazarr: v1.1.1
 - OS: Windows 10 21H2 (19044.2130)
 - Browser: Edge v106.0.1370.52 (Official build) (64-bit)

**Additional context**
Bazarr is running in a Docker container (linuxserver/bazarr:latest) on Synology DS920+, being viewed in Edge on Windows 10.

**Comment by halali** [comment]
Update bazarr, already fixed in 1.1.2

**Comment by yannduran** [comment]
OK thanks!

**Comment by yannduran** [comment]
Yes, that fixed the favicon problem, but after updating to `linuxserver/1.1.2` it seems to have broken LSIO's docker mods for using a `theme.park` theme. It gets ignored completely.

```
    environment:
      - DOCKER_MODS=gilbn/theme.park:bazarr
      - TP_THEME=plex
```

Going back to `linuxserver/1.1.1` and the theme works again. 

Strangely enough, when I was on v1.1.2, I discovered that clicking the button for setting Bazarr's dark mode works until the page is refreshed. It then goes back to whatever it was before button was clicked. Maybe related? The same seems to happen in v1.1.1, although it's not as noticeable as the plex theme is already a dark theme.

<img width="170" alt="22-10-23_16-46-12" src="https://user-images.githubusercontent.com/2031126/197376066-31941dd4-b7e2-49ad-8c6d-d4c016d499a3.png">

Something seems to be off with theming in v1.1.2

**Comment by halali** [comment]
But that is not an bazarr issue. We don't main theme park or linuxserver
images.

Dňa ne 23. 10. 2022, 7:49 Yann Duran ***@***.***> napísal(a):

**Comment by yannduran** [comment]
Except that it's highly unlikely that it's the fault of either theme.park or linuxserver, it's v1.1.2 itself that seems to have **caused** the problem. Between v1.1.1 & v1.1.2 neither of them are likely to have made any changes that would cause the issue. I mean why would they for just what's supposed to be a bug fix release (well, at least if you're following semantic versioning). It makes a lot more sense that the bug fix version has introduced a problem.

Did you even investigate before you fobbed it off as "not a Bazarr issue"???

**Comment by TRaSH-** [comment]
keep in mind 3rd party apps needs to make it working with any changes that a app does.
if a app decides for what ever reason to change a layout or what ever the 3rd party app needs to compensate for it.
So personally i actually start with asking in themepark support or what ever you use if more people run in to the issue and the dev of the 3rd party app can look in to the issue and see if he can fix it.

**Comment by yannduran** [comment]
Thanks for offering me your perspective. I agree with you 100% for a major version change (ie 2.0.0), or **maybe** even for a minor version (ie 1.3.0), even though I'd still class that as a regression bug, but I don't agree with that being the case for a bug fix version (ie 1.1.2). A bug fix release should not interfere with existing functionality.

Now if the guy had actually investigated **at all**, and had then said something like "we had to make some changes to some css files, maybe that's the problem" I would have appreciated that, and done exactly as you suggest.

It was the fact that the issue was immediately dismissed as "not Bazarr's problem" that I was addressing. FWIW other *rrs apps still work fine with theme.park themes.

**Comment by morpheus65535** [comment]
Let me put my perspective into that:
- We've never agreed to support any external themes.
- We do not take unsupported hack into account when developping Bazarr.
- We are free to improve UI as we see fit without it being a breaking change for unsupported hack.
- Other *rrs are free to support whatever usage they they want.
- Although halali's answer was succinct, it was based on fact: theme.park is unsupported so it's not Bazarr's problem.

Bazarr now include a light and dark theme. You should give it a try.
